sdks/go/test/integration/primitives/pardo.go

```diff
@@ -32,6 +33,9 @@ func init() {
 	register.Function3x2(asymJoinFn)
 	register.Function5x0(splitByName)
 	register.Function2x0(emitPipelineOptions)
+	beam.RegisterDoFn(reflect.TypeOf((*processElemBundleFinalizer)(nil)).Elem())
+	beam.RegisterDoFn(reflect.TypeOf((*finalizerInFinishBundle)(nil)).Elem())
+	beam.RegisterDoFn(reflect.TypeOf((*finalizerInAll)(nil)).Elem())
```

Comment:
This might be what is causing the check failures.

Reply:
I refactored to using the register funcs. Meanwhile converted PR to draft. I will explore how to filter out the tests for the docker execution.